drivers/spi/spi_ll_stm32.c

```diff
@@ -351,7 +351,7 @@ static int spi_stm32_configure(struct device *dev,
 	ll_func_set_fifo_threshold_8bit(spi);
 #endif
 
-#ifndef CONFIG_SOC_SERIES_STM32F1X
+#if !defined(CONFIG_SOC_SERIES_STM32F1X) && !defined(CONFIG_SOC_SERIES_STM32L1X)
```

Comment:
This is the wrong fix.  L1 has the FRF bit, you're just missing this fix in the HAL: https://github.com/zephyrproject-rtos/hal_stm32/pull/22 (enabled inside my other PR: https://github.com/zephyrproject-rtos/zephyr/pull/19729/commits/5e5afa0c5520dc80518ccaf2338271de76e92c6e)